aws-elastic-beanstalk/README.md

```diff
@@ -0,0 +1,34 @@
+Seekret
+=======
+An AWS elastic beanstalk deployment options.
+
+## Pre-requisites
+
+- Elastic beanstalk deployment over Amazon linux v2
+- Docker based beanstalk deployment.
+- ACCESS_KEY, SECRET_KEY and BUCKET_NAME (supported buckets: GCS / S3 / Azure Blob)
+
+## Deployment
+
+### Add elastic beanstalk hook files
+
+Seekret is using hook files to deploy our sniffer on your machine.
+1. [.ebextensions/options.config](options.config)
+   1. installs docker
```

Comment:
it will be installed if and only if the package does not exist on the machine (look at "Specifying versions" in https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/customize-containers-ec2.html#linux-packages)
added a comment for that